docs/images/architecture/k8s_deploy.svg

```diff
@@ -0,0 +1,369 @@
+       id="image12"
+       transform="scale(0.5,0.50163334)"
+       style="fill:#000000" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:18.66666603px;line-height:1.25;font-family:'Overpass Mono';-inkscape-font-specification:'Overpass Mono';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
+       x="139.17116"
+       y="353.49573"
+       id="text879"><tspan
+         sodipodi:role="line"
+         id="tspan877"
+         x="139.17116"
+         y="353.49573">$ kubectl gadget deploy</tspan></text>
```

Comment:
It seems to be this font is different than the one in "kubectl-gadget". Also, why is `kubectl-gadget` an image and not a text element?

Reply:
It's actually important to either embed the font or have the text elements converted to paths, because otherwise users that don't have the font installed will see the text rendered by some standard font that looks totally different.

For me right now it looks like this:
<img width="975" alt="Rendered SVG" src="https://user-images.githubusercontent.com/1550037/213325346-929ab1c2-9631-48ef-b62f-097fe7ab5273.png">